bitnami/mariadb/values.yaml

```diff
@@ -87,18 +87,17 @@ auth:
   ## Force users to specify required passwords
   ##
   forcePassword: false
-  ## Mount credentials as files instead of using an environment variable
-  ##
-  usePasswordFiles: false
```

Comment:
You are not deleting this on the values-production

Reply:
Fixed